### PR TITLE
I would like to add a reaction gif/image bot script thing

### PR DESCRIPTION
**Pull request by cavejay**
So the idea is that users can register and use images/gifs with the bot and then anyone can use the registered gifs/images.

It just means leggy could have a !mrw duck command that was a rubber duck.
Think 'bigger, free emotes'

atm the system could be abused and used to link to any resource or just repeat text. I'm going to look into enforcing links only and maybe somehow checking them to ensure validity.

tl;dr register text with bot and have the bot spit it back out as needed.

### CODE REVIEW

File: scripts/reaction.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// 
+
+module.exports = function (robot) {
+  robot.hear(/^!mrw (.+)/i, function (res) {
+	  var rStr = res.match[1];
+	  if(!rStr) return;
+	  robot.showReaction(rStr, res)
+  });
+  
+  robot.hear(/^uqcsbot new reaction (\w+\ [^ \n]+)/i, function (res) {
```
Comment (TRManderson) [inline scripts/reaction.js]: you can get rid of the ^uqcsbot part and just use robot.respond instead. This functions a lot better, because currently you can't PM uqcsbot things that start with ^ in the regex. Can you change that pls?

**Comment by cavejay** [comment]
updated with requested change
